website/templates/website/footer.html

```diff
@@ -0,0 +1,13 @@
+<div class="container" id="footerContainer">
+  <div class="row">
+    <div class="col-md-12 col-sm-12 col-xs-12 col-lg-12 footer-col">
+      <div id="footerContentContainer">
+        <div class="row">
+          <div class="col-md-12 col-sm-12 col-xs-12 col-lg-12">
+            <div>© Copyright 2008-2015, dipy developers neuroimaging@python.org. Created using Django and Bootstrap.</div>
```

Comment:
You can probably update the copyright year to 2016 :) Alternatively, it could probably be nice to automatically use the current year as the last year in the range...